projects/centre_console/src/relay_tx.c

```diff
@@ -0,0 +1,31 @@
+#include "relay_tx.h"
+#include "can_transmit.h"
+
+#define NUM_TX_RETRIES 5
+
+StatusCode relay_tx_init(RelayTxStorage *storage) {
+  CanTxRetryWrapperSettings retry_settings = { .retries = NUM_TX_RETRIES };
+  can_tx_retry_wrapper_init(&storage->can_retry_wrapper_storage, &retry_settings);
+  return STATUS_CODE_OK;
+}
+
+static void prv_tx_relay_state(CanAckRequest *ack_ptr, void *context) {
+  RelayTxStorage *storage = (RelayTxStorage *)context;
+  uint16_t relay_mask = 1 << storage->relay_id;
+  uint16_t relay_state = storage->relay_state << storage->relay_id;
+  CAN_TRANSMIT_SET_RELAY_STATES(ack_ptr, relay_mask, relay_state);
```

Comment:
I feel like you should make a macro for this, it'd make the previous one cleaner as well

Reply:
meh it's a test... you're right tho I'll clean it up in case I need to refactor my code then change this test...